docs/clients/python.md

```diff
@@ -76,36 +81,130 @@ Delete an environment configured on this machine.
 
 # cortex.client.Client
 
-## create\_api
-
-<!-- CORTEX_VERSION_MINOR -->
+## deploy
 
 ```python
- | create_api(api_spec: dict, handler=None, task=None, requirements=[], conda_packages=[], project_dir: Optional[str] = None, force: bool = True, wait: bool = False) -> list
+ | deploy(api_spec: Dict[str, Any], project_dir: str, force: bool = True, wait: bool = False)
 ```
 
-Deploy an API.
+Deploy an API from a project directory.
 
 **Arguments**:
 
 - `api_spec` - A dictionary defining a single Cortex API. See https://docs.cortex.dev/v/master/ for schema.
-- `handler` - A Cortex handler class implementation. Not required for TaskAPI/TrafficSplitter kinds.
-- `task` - A callable class/function implementation. Not required for RealtimeAPI/BatchAPI/TrafficSplitter kinds.
+- `project_dir` - Path to a python project.
+- `force` - Override any in-progress api updates.
+- `wait` - Streams logs until the APIs are ready.
+
+
+**Returns**:
+
+  Deployment status, API specification, and endpoint for each API.
+
+## deploy\_realtime\_api
+
+```python
+ | deploy_realtime_api(api_spec: Dict[str, Any], handler, requirements: Optional[List] = None, conda_packages: Optional[List] = None, force: bool = True, wait: bool = False) -> Dict
+```
+
+Deploy a Realtime API.
+
+**Arguments**:
+
+- `api_spec` - A dictionary defining a single Cortex API. See https://docs.cortex.dev/v/master/workloads/realtime-apis/configuration for schema.
```

Comment:
we should add a comment in the vicinity of this line and for each docs link below like we have for the other one (`<!-- CORTEX_VERSION_MINOR -->`) - this helps me find which changes to make when doing releases. It's a little hacky, but we do it in `dev/generate_python_client_md.sh`.